example/simple_fetch_content/simple.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <c_bindings/export.hpp>
```

Comment:
Is c_bindings the right folder name? Should it be maybe `gt_cbinding` or whatever? Or should we change the macro prefix `GT_`? These are just questions - I have no answer to them, but it feels a bit inconsistent.

Reply:
It is inconsistent because I am waiting for a new name. I started by changing `gridtools/c_bindings` to `c_bindings` in the include path, but then didn't remove the `GT` everywhere because it is quite a lot of things to rename...